mycore-base/src/main/java/org/mycore/common/events/MCRShutdownHandler.java

```diff
@@ -159,6 +161,30 @@ void runClosables() {
         }
     }
 
+    /**
+     * only used to make {@link MCRShutdownHandler#resetCloseables()} testable
```

Comment:
Javadoc comment is ambiguous. It should explain that this method (and the following) should only be called by a test framework and are not for applications to use.

Reply:
is fixed